CryptoPkg/CryptoPkg.dsc

```diff
@@ -115,8 +115,10 @@
   RngLib|MdePkg/Library/BaseRngLib/BaseRngLib.inf
 
 [LibraryClasses.ARM, LibraryClasses.AARCH64]

```

Comment:
nit: commit name still references StandaloneMmPkg.

Reply:
Updated in latest push.